cmd/applyconfig/applyconfig.go

```diff
@@ -239,6 +244,57 @@ func applyConfig(rootDir, cfgType string, process processFn) error {
 	return nil
 }
 
+type censoringFormatter struct {
+	sync.RWMutex
+	secrets  sets.String
+	delegate logrus.Formatter
+}
+
+func (f *censoringFormatter) Format(entry *logrus.Entry) ([]byte, error) {
```

Comment:
Feels bad to copy this. Can't we factor out the difference in behavior (impl of how to censor) and reuse the logging bits

Reply:
Understood.
Will work on this for upstream today, following https://github.com/openshift/ci-tools/pull/224#discussion_r337089805